include/astshim/PolyMap.h

```diff
@@ -225,7 +225,7 @@ friend class Object;
         std::vector<double> const & lbnd,
         std::vector<double> const & ubnd
     ) const {
-        int desSize = forward ? getNin() : getNout();
+        auto const desSize = static_cast<unsigned int>(forward ? getNin() : getNout());
```

Comment:
I see your point. However, the number of axes will always be small, and unsigned integers are a nuisance when taking differences, so I prefer to stick to `int` where practical. If we do decide to change this there are many other `get` methods that would make sense to return unsigned ints (and some `set` methods that would take unsigned ints).